napalm/eos/eos.py

```diff
@@ -966,7 +966,7 @@ def get_arp_table(self, vrf=""):
             interface = str(neighbor.get("interface"))
             mac_raw = neighbor.get("hwAddress")
             ip = str(neighbor.get("address"))
-            age = float(neighbor.get("age"))
+            age = float(neighbor.get("age", 0.0))
```

Comment:
Should we use `-1.0` to mark static entries?  I notice nxos uses this approach, but IOS doesn't.  Perhaps worth standardizing on?

Reply:
I would vote for -1.0 for all OSes, as looks like "-1" is a standard value for N/A in Napalm.

Reply:
Agreed. @praniq - please feel free to update this branch, then I can merge it.

Reply:
Updated EOS, IOS and IOS XR. Please have a look. Thanks